datasets.py

```diff
@@ -367,10 +367,10 @@ def setup_anns(self):
                             biggest_bbox = this_bbox
 
                 if m_presence > 0 and w_presence == 0:
-                    self.anns[key] = [self.anns[key], [2, biggest_bbox], [0]]
+                    self.anns[key] = [self.anns[key], [[2], [biggest_bbox]], [0]]
```

Comment:
given the recent changes in attribute_analysis, does 2 and 1 make sense here? or should they be changed to 0 and 1, or whatever coco is

Reply:
also, with the new attribute_based code, do you need to define some sort of self.attribute_names for all of these classes?